ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/page/wizard/project/NewLiferayModulesExtWizard.java

```diff
@@ -27,8 +28,28 @@ public NewLiferayModulesExtWizard(SWTBot bot) {
 		super(bot, 2);
 	}
 
+	public void clickBrowseBtn() {
+		getBrowseBtn().click();
+	}
+
+	public void clickDeleteBtn() {
+		getDeleteBtn().click();
+	}
+
+	public ToolbarButtonWithTooltip getAddOriginMoudleBtn() {
+		return new ToolbarButtonWithTooltip(getShell().bot(), ADD_FILES_FROM_ORIGINAL_MODULE_TO_OVERRIDE);
+	}
+
 	public ToolbarButtonWithTooltip getBrowseBtn() {
 		return new ToolbarButtonWithTooltip(getShell().bot(), BROWSE, 1);
 	}
 
+	public ToolbarButtonWithTooltip getDeleteBtn() {
+		return new ToolbarButtonWithTooltip(getShell().bot(), DELETE);
+	}
+
+	public Text getOriginalModule() {
+		return new Text(getShell().bot(), ORIGINAL_MODULE_NAME);
```

Comment:
try to get the original original module instead of Blank, otherwise, we won't add test code for this